package.json

```diff
@@ -145,7 +145,9 @@
     "not dead"
   ],
   "author": "",
-  "dependencies": {},
+  "dependencies": {
+    "html-entities": "^1.3.1"
```

Comment:
@yukienomiya could we move this to `devDevependencies` please?
(thanks @limzykenneth for noticing this!)

Reply:
Done! Thanks @limzykenneth :)